cmd/root.go

```diff
@@ -34,19 +39,27 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initConfig)
 
-	RootCmd.PersistentFlags().StringVar(&sfdcUsername, "username", "", "Salesforce Username")
-	RootCmd.PersistentFlags().StringVar(&sfdcPassword, "password", "", "Salesforce Password")
-	RootCmd.PersistentFlags().StringVar(&appClientID, "client-id", "", "Connected App Client ID")
-	RootCmd.PersistentFlags().StringVar(&appClientSecret, "client-secret", "", "Connected App Client Secret")
-	RootCmd.PersistentFlags().StringVar(&apiVersion, "api-version", "36.0", "Salesforce API Version")
-	RootCmd.PersistentFlags().StringVar(&sfdcHost, "host", "https://login.salesforce.com", "Salesforce Login Url")
+	RootCmd.PersistentFlags().StringVar(&appClientID, "client-id", "", "OAuth Client ID")
+	RootCmd.PersistentFlags().StringVar(&appClientSecret, "client-secret", "", "OAuth Client Secret")
+	RootCmd.PersistentFlags().StringVar(&apiVersion, "api-version", "", "API Version")
+	RootCmd.PersistentFlags().StringVar(&host, "host", "",
+		"API Host base URL, e.g. my-site.skuidsite.com for Skuid Platform or my-domain.my.salesforce.com for Salesforce")
+	RootCmd.PersistentFlags().StringVarP(&module, "module", "m", "", "Module name(s), separated by a comma.")
+	RootCmd.PersistentFlags().StringVarP(&password, "password", "p", "", "Skuid Platform / Salesforce Password")
+	RootCmd.PersistentFlags().StringVarP(&targetDir, "dir", "d", "", "Input/output directory.")
```

Comment:
The third parameter is the default. I noticed you had a place where you checked if it was `""`. If you set it here, you won't need to do that.